seqr/views/utils/export_utils_2_3_tests.py

```diff
@@ -14,23 +16,24 @@ class ExportTableUtilsTest(TestCase):
 
     def test_export_table(self):
         header = ['column1', 'column2']
-        rows = [['row1_v1', 'row1_v2'], ['row2_v1', 'row2_v2']]
+        rows = [['row1_v1\xe2', 'row1_v2'], ['row2_v1', 'row2_v2']]
 
         # test tsv format
         response = export_table('test_file', header, rows, file_format='tsv')
 
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.get('content-disposition'), 'attachment; filename="test_file.tsv"')
-        self.assertEqual(response.content, '\n'.join(['\t'.join(row) for row in [header]+rows]) + '\n')
+        self.assertEqual(response.content.decode('utf-8', errors = 'ignore'),
```

Comment:
I don't want to test that the decoded content is decoded, I want to test that the raw content is decoded. This needs to not change

Reply:
Changed to encode the expected data.